build/dockerfiles/rollup_relayer.Dockerfile

```diff
@@ -15,12 +15,27 @@ RUN go mod download -x
 # Build rollup_relayer
 FROM base as builder
 
+RUN mkdir -p /scroll/lib/
+RUN apt-get -qq update && apt-get -qq install -y wget
+RUN wget -O /scroll/lib/libzktrie.so https://github.com/scroll-tech/da-codec/releases/download/v0.0.0-rc0-ubuntu20.04/libzktrie.so
```

Comment:
Will we need to keep this verison and the version of `scroll-tech/da-codec` in sync? Or do you expect that the Rust code will not change often, even if we change the Go part of the codec?

Reply:
yeah. once libzstd is updated, need to update the version as well. I think @HAOYUatHZ 's suggestion to make it a variable sounds good.

Reply:
a similar mechanism, as upgrading libzkp version.